cumulusci/tasks/github/merge.py

```diff
@@ -211,6 +212,10 @@ def _is_release_branch(self, branch_name):
         """A release branch begins with the given prefix"""
         return is_release_branch(branch_name, self.options["branch_prefix"])
 
+    def _get_base_branch(self, branch_name):
+        """Strip branch_prefix from branch name, returning the base branch"""
```

Comment:
"base branch" is the term github uses to refer to the target branch of a pull request. Can we call this something else, maybe "_get_release_number" since it's only called from places that then `int()` it